agent_build/linux/scalyr-agent-2-config

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
```

Comment:
That's here for backward compatibility reasons, right?

Reply:
Yes.  I think there's a high chance that some of the users use it during their upgrades or something else.